ablator/modules/loggers/main.py

```diff
@@ -164,6 +248,15 @@ def update(
         metrics: Union[TrainMetrics, dict],
         itr: Optional[int] = None,
     ):
+        """ Update the dashboard with the given metrics.
```

Comment:
it also writes some metrics to json files and update the current metadata(log_iteration)